### ISSUE TITLE
Inconsistent load reporting Zone/Sys vs Window AC and DX Coil (ref: CR7092) (CR #7100)

### ISSUE THREAD

**Issue by axelstudios** [NREL/EnergyPlus#1886]
## 
#### Description

DShirey 22 Sept 2006
Here is the variable being calculated (SNLOAD)... from ZoneTempPredictorCorrector.f90

```
   ! SNLOAD is the single zone load, without Zone Multiplier or Zone List Multiplier
   SNLOAD = ZoneEnthalpyIn - (Node(ZoneNodeNum)%MassFlowRate / ZoneMult) * CpAir * Node(ZoneNodeNum)%Temp &
          + NonAirSysEquipHeatAdd(ZoneNum) / ZoneMult
```

The difference with zone/sys sensible cooling reports, isy 55W during "pulldown" and then the difference goes to <2 Watts.

[WFB 28Feb2007]

Finally, something I can answer! As described in CR 7092, the issue here is why doesn't
the unit output (unit = Window AC) match the zone sensible load (SNLOAD, or 
Zone/Sys Sensible Cooling Rate as a report variable). The answer is that SNLOAD is the
corrected load (using recalc zone temp after the system has run). The unit output should
match the predicted load (Zone/Sys Sensible Load Predicted), which it does (see results
for Zone 1 / Resistive zone in defect file 7100-WindACAuto). In the defect file I've zeroed
out the outside air and (almost) the fan heat. The predicted load matches the unit
sens output very well except in the first few detailed time steps of the pull down when
the predicted load is greater than the unit capacity. 

In summary, the mismatch between SNLOAD and unit output due to our predictor - 
corrector calculation method. It is nothin to be concerned about.
## 

Addressed by F Buhl
Last build tested: `06.09.22 V1.4.0.019`
